src/Apps/Order/Routes/Submission/index.tsx

```diff
@@ -50,11 +50,7 @@ export class SubmissionRoute extends Component<SubmissionProps> {
             }
             Sidebar={
               <Flex flexDirection="column">
-                <Placeholder
-                  height="180px"
-                  name="Shipping and payment details"
-                  mb={xs ? 2 : 3}
-                />
+                <ShippingAndPaymentDetails order={order} mb={xs ? 2 : 3} />
```

Comment:
By actually reading the docs I discovered that these styled-system props can be specified with arrays to get responsive styles. So `xs ? 2 : 3` would be just `[2, 3]` I think I'll fix all of these up in a separate PR.